src/features/app_notification/index.tsx

```diff
@@ -10,7 +10,11 @@ const AppNotification = () => {
 
   return (
     <>
-      <ButtonIcon onClick={handleOpenNotification}>
+      <ButtonIcon
+        onClick={handleOpenNotification}
```

Comment:
@FussuChalice: for this change, my idea would be to just toggle the state value. This should be change on the `useAppNotification.tsx` file:

```
const handleToggleNotificationState = async () => {
    setIsMyAssignmentOpen(false);

    setOpen((prev) => !prev);
};
```

Also, could you revert all changes in the `IconButton` component about the background color handling? At this moment, I do not see yet the need to change it.

Thanks.

Reply:
Yes, sure.